src/Sql/dbo/Functions/PolicyApplicableToUser.sql

```diff
@@ -25,11 +25,12 @@ LEFT JOIN
 WHERE
     (
         (
-            OU.[Status] > 0
+            OU.[UserId] IS NOT NULL     -- OrgUsers who have accepted their invite and are linked to a UserId
```

Comment:
This becomes redundant given the check on `@UserId` equality below it, perhaps the more appropriate check would be:
```sql
OU.[Status] != 0
```
Since we're going both directions away from zero in this check, we essentially should only exclude zero.

Reply:
Done